examples/simple-udp/gameserverallocation.yaml

```diff
@@ -12,14 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#
+# A GameServerAllocation against a Fleet named 'simple-udp'
 # (GameServerAllocation is currently experimental, and likely to change in upcoming releases)
+#
 
-apiVersion: "stable.agones.dev/v1alpha1"
+apiVersion: "allocation.agones.dev/v1alpha1"
```

Comment:
general question - is allocation the only thing that will be in this API going forward? Perhaps we need a broader name to indicate "services that external consumers are going to call".

Reply:
That's a really interesting question. Groups are pretty broad, and we could definitely expand this - and anything that is "external" would likely need to be managed by multiple pods to ensure HA (as discussed above). Which would give us an internal delineation - anything under this group needs to be managed by multiple pods, anything in the `stable` group, is managed by the controller (which self heals).

Maybe `external.agones.dev` ?

It also begets a question of is "stable" the right name for the original group. I'm starting to wonder if `core.agones.dev` would have been better? Or something else?

Definitely open to suggestions here.

Reply:
Looking at #703 - I'm feeling like `allocation` is actually the right choice here for the group name.